src/rt/aaA.d

```diff
@@ -152,6 +152,21 @@ private:
         deleted = 0;
         GC.free(obuckets.ptr); // safe to free b/c impossible to reference
     }
+
+    void clear() pure nothrow
+    {
+        foreach (ref b; buckets[firstUsed .. $])
+        {
+            if (b.filled)
+            {
+                b.hash = HASH_DELETED;
```

Comment:
> just memset(0) the whole bucket array

OK, I wasn't sure if I was missing something on the HASH_DELETED record keeping, so I played it safe. I can use memset instead.